controllers/controllers.technicians.js

```diff
@@ -20,3 +21,15 @@ exports.getTechnician = async (req, res, next) => {
     next(e);
   }
 };
+
+exports.patchTechnician = async (req, res, next) => {
+  const updates = req.body;
+  const id = req.params.user_id;
+  try {
+    await updateTechnician(id, updates)
+    const updatedTechnician = await findTechnician(id);
+    res.status(200).send(updatedTechnician)
```

Comment:
here we should send `{updatedTechnician}` and destructure it in our test like `const { updatedTechnician } = body;`. Also consider changing the name to just `technician`